### ISSUE TITLE
Bitcode will be dropped with Xcode 7

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gzog/analytics-issues#671]
```
---------------------------------------------------------------------------
NOTE: This issue tracking system is for Google Analytics developer products
only.
If you are not a developer/programmer visit:
http://www.google.com/analytics/support.html
---------------------------------------------------------------------------
Name of affected component: Mobile Tracking

Name of related library and version, if applicable (e.g. Android, iOS,
Snippets, etc.): iOS

Issue summary:
Provide a brief summary of the issue you're experiencing.

Steps to reproduce issue:
1.  Use Xcode 7.  In your project ENABLE_BITCODE is on by default.
2.  Build an app using the Google Analytics iOS library.
3.

Expected output:
What do you expect to see after performing the above steps?
No warnings

Actual results:
What do you actually see after performing the above steps?
These warnings (This will be an error in the future.):

ld: warning: -weak_framework is treated as -framework when used with 
-bitcode_bundle (Xcode setting ENABLE_BITCODE=YES). Weak linking may still 
happen based on availability mark-up in headers
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAI.o)' was built without bitcode. You must 
rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an 
updated library from the vendor, or disable bitcode for this target. Note: This 
will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIDictionaryBuilder.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIFields.o)' was built without bitcode. You 
must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an 
updated library from the vendor, or disable bitcode for this target. Note: This 
will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIUncaughtExceptionHandler.o)' was built 
without bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIBatchingDispatcher.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIDataStore.o)' was built without bitcode. 
You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain 
an updated library from the vendor, or disable bitcode for this target. Note: 
This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIDefaultLogger.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIEcommerceFields.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIStringUtil.o)' was built without bitcode. 
You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain 
an updated library from the vendor, or disable bitcode for this target. Note: 
This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAITrackerImpl.o)' was built without bitcode. 
You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain 
an updated library from the vendor, or disable bitcode for this target. Note: 
This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIURLConnection.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIUsageTracker.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAICampaign.o)' was built without bitcode. 
You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain 
an updated library from the vendor, or disable bitcode for this target. Note: 
This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAICoreDataUtil.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIExceptionParser.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIHitUtil.o)' was built without bitcode. You 
must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an 
updated library from the vendor, or disable bitcode for this target. Note: This 
will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIReachabilityChecker.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIRequestBuilder.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAITrackerModel.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIUtil.o)' was built without bitcode. You 
must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an 
updated library from the vendor, or disable bitcode for this target. Note: This 
will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAIHitFormat.o)' was built without bitcode. 
You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain 
an updated library from the vendor, or disable bitcode for this target. Note: 
This will be an error in the future.
ld: warning: URGENT: all bitcode will be dropped because 
'.../libGoogleAnalyticsServices.a(GAICompressionUtil.o)' was built without 
bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. Note: This will be an error in the future.

Notes:
Provide any additional information which might be useful here. Feel free to
attach screenshots or sample code which demonstrates the issue being
described.

```

Original issue reported on code.google.com by `ellucian...@gmail.com` on 18 Jun 2015 at 3:12

**Comment by GoogleCodeExporter** [comment]
```
I also needed the BITCODE support in my iOS App.
```

Original comment by `BestC...@gmail.com` on 26 Aug 2015 at 2:27

**Comment by GoogleCodeExporter** [comment]
```
In XCode 7 Beta 5 I get the following compile error with ENABLE_BITCODE=YES in 
build settings.

libGoogleAnalyticsServices.a(TAGDataProvider.o)' does not contain bitcode. You 
must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an 
updated library from the vendor, or disable bitcode for this target. for 
architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Original comment by `onche...@gmail.com` on 28 Aug 2015 at 8:20

**Comment by GoogleCodeExporter** [comment]
```
Ditto
```

Original comment by `neil.s.j...@gmail.com` on 31 Aug 2015 at 6:36

**Comment by GoogleCodeExporter** [comment]
```
Same error as described by #2 
```

Original comment by `vbergae` on 2 Sep 2015 at 6:39

**Comment by GoogleCodeExporter** [comment]
```
Same issue here with Xcode Version 7.0 beta 6 (7A192o).
```

Original comment by `silentas` on 4 Sep 2015 at 8:48

**Comment by GoogleCodeExporter** [comment]
```
Gold master is out... 
```

Original comment by `RRomanc...@gmail.com` on 10 Sep 2015 at 5:17

**Comment by GoogleCodeExporter** [comment]
```
Come on, the Xcode 7 GM is out! We need the BITCODE support!
```

Original comment by `valeriolg` on 10 Sep 2015 at 11:52

**Comment by GoogleCodeExporter** [comment]
```
This is the same as with 64bit support. Somehow it takes ages to rebuild the 
library. This is my last project with GA.
```

Original comment by `ed...@cybionic-systems.com` on 10 Sep 2015 at 11:55

**Comment by GoogleCodeExporter** [comment]
```
Looks like only Google Analytics still doesn't support. Other used analytics 
tools like Flurry, Heap, Amazon, Yandex, Fabric are fully support bitcode.
```

Original comment by `kirill.k...@gmail.com` on 10 Sep 2015 at 11:56

**Comment by GoogleCodeExporter** [comment]
```
Would be nice to have BITCODE support! 
Thanks
```

Original comment by `miroslav...@rohlik.cz` on 10 Sep 2015 at 2:16

**Comment by GoogleCodeExporter** [comment]
```
yep - we need this lib updated!
```

Original comment by `jose.go...@saplinglearning.com` on 10 Sep 2015 at 5:05

**Comment by GoogleCodeExporter** [comment]
```
Met the same issue. Need bitcode support.
```

Original comment by `derjo...@gmail.com` on 11 Sep 2015 at 1:38

**Comment by GoogleCodeExporter** [comment]
```
When do you release new version Dev team???
```

Original comment by `ltg...@gmail.com` on 11 Sep 2015 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
It's both amusing and sad how similar this case is to the 64-bit support. I 
believe It took Google months to even respond to the issue, and the support was 
eventually added almost a year later. It is abundantly clear that iOS is not a 
priority to the Analytics team.

Please note that bitcode support is optional unless you are building watchOS 
apps. If you are, I recommend you switch to another analytics platform.
```

Original comment by `mar...@essentia.fi` on 11 Sep 2015 at 6:31

**Comment by GoogleCodeExporter** [comment]
```
Right, Google is too slow to release new features supported by Apple and give 
users better expirience in apps.
```

Original comment by `kirill.k...@gmail.com` on 11 Sep 2015 at 6:35

**Comment by GoogleCodeExporter** [comment]
```
What is this Google company you speak of... Alphabet Inc right?  Yeah this is 
the only thing holding us up from doing iOS9 releases.
```

Original comment by `michael....@healthpartners.com` on 11 Sep 2015 at 2:38

**Comment by GoogleCodeExporter** [comment]
```
Need it too!
```

Original comment by `obr...@gmail.com` on 14 Sep 2015 at 1:34

**Comment by GoogleCodeExporter** [comment]
```
.... how do i get around this?
```

Original comment by `takombor...@gmail.com` on 14 Sep 2015 at 3:29

**Comment by GoogleCodeExporter** [comment]
```
workaround...in project settings, build settings, under build options, select 
NO for Enable Bitcode. NOTE: this wont work if your app has uses watchOS
```

Original comment by `jose.go...@saplinglearning.com` on 14 Sep 2015 at 3:30

**Comment by GoogleCodeExporter** [comment]
```
FYI, with the Xcode 7 GM Seed you can turn off bitcode on the phone and enable 
it on the watch. This allows the use of Google Analytics API on the phone.

Would be nice to add it to the watch too!
```

Original comment by `ellucian...@gmail.com` on 14 Sep 2015 at 10:51

**Comment by GoogleCodeExporter** [comment]
```
If bitcode support is added, it will not work with Xcode 6. I'm pretty sure 
that's the reason Google is not coming up with bitcode supported lib. 
```

Original comment by `gau...@moengage.com` on 15 Sep 2015 at 9:41

**Comment by GoogleCodeExporter** [comment]
```
Maybe it's a good idea if google made a iOS 8 dynamic framework and switch 
bitcode on for the framework. How hard would it be for them add this extra 
build target to their project.
```

Original comment by `ed...@cybionic-systems.com` on 15 Sep 2015 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
Xcode 7.0 now released, please add bitcode support...
```

Original comment by `rock88a@gmail.com` on 17 Sep 2015 at 6:55

**Comment by GoogleCodeExporter** [comment]
```
I've just update to Xcode 7.0 and getting the same issue. Please update the 
Google Analytics Library.
```

Original comment by `lephukha...@gmail.com` on 17 Sep 2015 at 7:45

**Comment by GoogleCodeExporter** [comment]
```
We need support of bitcode in this lib. It's urgent thing.
```

Original comment by `Yanke...@gmail.com` on 17 Sep 2015 at 8:56

**Comment by GoogleCodeExporter** [comment]
```
When does it support bitcode? We really need it!
```

Original comment by `junya...@gmail.com` on 18 Sep 2015 at 10:21

**Comment by GoogleCodeExporter** [comment]
```
Yeah, add bitcode support!
```

Original comment by `matejuk...@gmail.com` on 18 Sep 2015 at 4:25

**Comment by GoogleCodeExporter** [comment]
```
As usual, its the GA library holding up iOS upgrades. *FacePalm*
```

Original comment by `sanj...@zillow.com` on 18 Sep 2015 at 5:36

**Comment by GoogleCodeExporter** [comment]
```
Would love to see this ASAP! 
```

Original comment by `M.V.Mull...@gmail.com` on 18 Sep 2015 at 11:08

**Comment by GoogleCodeExporter** [comment]
```
Pls. add this soon
```

Original comment by `vicky.me...@gmail.com` on 21 Sep 2015 at 6:25

**Comment by GoogleCodeExporter** [comment]
```
Please add it!!!
```

Original comment by `wie...@gmail.com` on 21 Sep 2015 at 4:32

**Comment by GoogleCodeExporter** [comment]
```
Currently unable to submit an update to the appStore for iOS9 as BITCODE has 
not been enabled on the library.  What is the ETA to make this happen?  
Alternatively, will you provide the libraries as full open source to allow the 
community to fix these issues please?
```

Original comment by `Jason.Br...@googlemail.com` on 21 Sep 2015 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
you can still submit apps if you disable bitcode option in your project, for 
now. 
```

Original comment by `jose.go...@saplinglearning.com` on 21 Sep 2015 at 9:50

**Comment by GoogleCodeExporter** [comment]
```
Not necessarily. Xcode has a frustrating code signing bug when a parent app 
disables bit code but embedes a watch target that requires bit code to be 
enabled. C.f: https://forums.developer.apple.com/thread/10392

I'm trying to update my dependencies to newer builds that support bit code. It 
seems Google Analytics is the last dependency that does not support bit code.
```

Original comment by `bran...@acorns.com` on 21 Sep 2015 at 9:52

**Comment by GoogleCodeExporter** [comment]
```
Unless one wants to submit for watchOS...

P

21. 9. 2015 v 23:50, analytics-issues@googlecode.com:
```

Original comment by `obr...@gmail.com` on 21 Sep 2015 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
As long as you don't include the google analytics library in your watch target, 
you can leave bitcode on for the watch. Then on the phone target turn off 
bitcode.

This way you can submit your app which includes watch and analytics in the 
phone.

The downside is you can't utilize analytics in watch code and your phone 
application can not participate in app thinning.
```

Original comment by `b...@brethansenfamily.com` on 22 Sep 2015 at 4:49

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Just adding my +1 to the noise. Such a pity that their iOS team is notorious 
for their silence. Whilst I love the recommendation for "Not using Google for 
Analytics" the companies we work for aren't as progressive. What I plan on 
doing is using GA along with something else like MixPanel and try to convince 
the business to make the switch once they can see how much better they are. I 
think we can all agree that the less iOS libraries from Google your product 
relies on the better.
```

Original comment by `danielhg...@gmail.com` on 22 Sep 2015 at 2:46

**Comment by GoogleCodeExporter** [comment]
```
Come on Google, fix this! I'm dropping google analytics and will roll my own 
analytics code for a future app release. EVERY time I want to release a new 
version of my app I am delayed by some issue or other with google analytics. 
Forget it. My users will no doubt be happy to hear that I'm not farming their 
device data for google anymore too! Bonus!
```

Original comment by `i...@myhourz.com` on 24 Sep 2015 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
How long should we still waiting?!
```

Original comment by `rusya....@gmail.com` on 24 Sep 2015 at 1:15

**Comment by GoogleCodeExporter** [comment]
```
Google, anything ...
Please provide a new version of Google Analytics with bitcode enabled.
```

Original comment by `milen.ha...@mentormate.com` on 24 Sep 2015 at 1:32

**Comment by GoogleCodeExporter** [comment]
```
Google PleaaaseHelppp!
```

Original comment by `fabrice....@gmail.com` on 25 Sep 2015 at 11:43

**Comment by GoogleCodeExporter** [comment]
```
It seems to me obvious that mr. G. is not interested in us: in 3 months they 
even did not borther to post any response. I am dropping Goggle Analytics 
support from my apps once and forever and I suggest you the same.
```

Original comment by `Janos.To...@gmail.com` on 25 Sep 2015 at 4:36

**Comment by GoogleCodeExporter** [comment]
```
Is there any news about when this will be fixed?
```

Original comment by `christia...@codecentric.de` on 29 Sep 2015 at 7:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `mcoh...@google.com` on 29 Sep 2015 at 3:27
- Changed state: **Prioritized**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
fuck
```

Original comment by `Nikita.A...@gmail.com` on 30 Sep 2015 at 1:52

**Comment by GoogleCodeExporter** [comment]
```
Uh, no. It should be added. If not, maybe we should drop something else
from our apps.
```

Original comment by `h.bruin...@coolblue.nl` on 30 Sep 2015 at 1:55

**Comment by GoogleCodeExporter** [comment]
```
Don't hold your breath. Last time there was Google Sign In issue preventing 
apps from being accepted in the app store. 
(https://code.google.com/p/google-plus-platform/issues/detail?id=900)  It took 
Google 10 months to fix it. 10 months to fix show stopper that prevented from 
using Google's iOS library. 10 months. Only 3 months passed since this issue 
was opened.
```

Original comment by `roman.r...@gmail.com` on 30 Sep 2015 at 4:00

**Comment by GoogleCodeExporter** [comment]
```
As always, Google is quicker in copying Apple than supporting Apple.

I had the impression that Google was the open(source) company. Why just not 
make the library opensource.? Is their something we're not allowed to see? Is 
Google the Volkswagen of the IT world?

```

Original comment by `ed...@cybionic-systems.com` on 30 Sep 2015 at 4:08

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
It's clear that Google is not really interested in improving/updating this 
library. Since version 3.0.8 at least it's been giving me problems in apps 
submission because it contained "unsupported or illegal" instructions. I had 
already switched to other analytics libraries out there, and I would definitely 
suggest you to do the same.
```

Original comment by `cont...@digitalleaves.com` on 30 Sep 2015 at 4:27

**Comment by GoogleCodeExporter** [comment]
```
As slicing currently isn't working 
(https://developer.apple.com/news/?id=092415a&1443125762), why don't you 
disable bitcode for now and submit a new version of your apps later when Google 
has fixed analtycs?

```

Original comment by `fno...@gmail.com` on 30 Sep 2015 at 6:18

**Comment by GoogleCodeExporter** [comment]
```
Turning off bitcode for watch is not an option. It has to be enabled
```

Original comment by `b...@brethansenfamily.com` on 30 Sep 2015 at 6:39

**Comment by GoogleCodeExporter** [comment]
```
The closed source status of this project mixed with blatant disregard for 
modernization is very alarming. Regardless of the current status of app 
slicing, bitcode support is a matter of changing a single boolean before 
building. Both 64bit support and now bitcode took months and months of 
strongarming and threats from the community for them to finally cut a release. 
Apple gave us all half a year (even longer for arm64 support) to make these 
changes, and google can't even rebuild a static library with a single build 
setting change. 

Does anyone on the ios SDK team actually build ios apps? Maybe you can ask an 
intern to ENABLE_BITCODE=YES, and cut a new release? There is absolutely no 
reason we should be blocked on something like this right now. Are internal 
teams like inbox not using the GA client? Do they have access to internal 
source? I'm surprised other internal ios projects within google have not taken 
this issue into their own hands.  
```

Original comment by `RRomanc...@gmail.com` on 1 Oct 2015 at 7:47

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This did not update the entirety of the Google SDK, and specifically, Google 
Analytics under which this project is filed.

Installing Google 1.1.0 (was 1.0.7)
Using GoogleAnalytics (3.13.0)

> '/Pods/GoogleUtilities/Libraries/libGTM_NSData+zlib.a(GTMNSData+zlib.o)' does 
not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. for architecture arm64

```

Original comment by `ataran...@gmail.com` on 1 Oct 2015 at 8:26

**Comment by GoogleCodeExporter** [comment]
```
Hey guys (GA team),

I'm a completely new user to this library and I just wanted to let you know, 
due to the overwhelming warning here in this thread from responsible (+awesome) 
developers, I am steering clear of Google's entire iOS (Swift/Obj-C) offerings.

We are choosing not to use Cloud Messaging, Sign In, and the whole rest of it 
due to this issue. High priority does not even begin to cover it. RRoman above 
is correct - it's one simple build flag change and we're done with you over it.

-a potential customer

```

Original comment by `s...@momentum.io` on 1 Oct 2015 at 11:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `mcoh...@google.com` on 2 Oct 2015 at 12:56

**Comment by GoogleCodeExporter** [comment]
```
What are you doing, GA Team???
```

Original comment by `ltg...@gmail.com` on 2 Oct 2015 at 8:41

**Comment by GoogleCodeExporter** [comment]
```
Google PLS  (╯°□°)╯︵ ┻━┻
```

Original comment by `miroslav...@rohlik.cz` on 2 Oct 2015 at 8:43

**Comment by GoogleCodeExporter** [comment]
```
Google A Guys, as said by many, it is just a matter of recompiling with the 
flag set correctly, PLEASE take 5 minutes to do this. 
```

Original comment by `a...@wapolabs.com` on 2 Oct 2015 at 12:23

**Comment by GoogleCodeExporter** [comment]
```
Just to help the Google guys along, the Open Source community recently got 
CardIO's spec to work with Bitcode. If the GA guys are struggling they could 
follow the thread here https://github.com/card-io/card.io-iOS-SDK/issues/119 
and see how CardIO managed to resolve this.
```

Original comment by `danielhg...@gmail.com` on 2 Oct 2015 at 1:58

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Pretty disappointed at the utter lack of care in fixing this. It should have 
been done months ago. Given it's still outstanding, there should be an engineer 
working on this right now until it is released.
```

Original comment by `jamesebi...@gmail.com` on 2 Oct 2015 at 2:19

**Comment by GoogleCodeExporter** [comment]
```
Maybe Google are moving out of the iOS mobile analytics game?  Or maybe their 
coder is taking a sabbatical away from coding....

Maybe the title of this issue needs changing to highlight that BITCODE is 
preventing Google Analytics being deployed to iOS app that support Apple Watch?
```

Original comment by `Jason.Br...@googlemail.com` on 2 Oct 2015 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
I tried their new pod 'Google/Analytics' thing too. It has the same bitcode 
problem with some utility it depends on. 
```

Original comment by `a...@wapolabs.com` on 2 Oct 2015 at 3:00

**Comment by GoogleCodeExporter** [comment]
```
Please fix this issue!!! I am also developing a watchos2 app!!
```

Original comment by `wie...@routing4you.com` on 5 Oct 2015 at 7:49

**Comment by GoogleCodeExporter** [comment]
```
Is there any updates on this issue? I'm getting the same error as #57 :
> '/Pods/GoogleUtilities/Libraries/libGTM_NSData+zlib.a(GTMNSData+zlib.o)' does 
not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting 
ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode 
for this target. for architecture arm64
```

Original comment by `ke...@pinch.eu` on 5 Oct 2015 at 3:55

**Comment by GoogleCodeExporter** [comment]
```
"App Slicing Now Available" (https://developer.apple.com/news/)

Come on Google we're waiting.... Or do we have to wait until you 'copy' slicing 
into Android?
```

Original comment by `ed...@cybionic-systems.com` on 6 Oct 2015 at 5:52

**Comment by GoogleCodeExporter** [comment]
```
This is holding up my watchos2 app release. 
```

Original comment by `twal...@gmail.com` on 6 Oct 2015 at 11:02

**Comment by GoogleCodeExporter** [comment]
```
To submit watchOS 2 apps and still have GA in the project, all you have to do 
is enable bitcode for the watch app and watch extension targets, and disable 
bitcode for the iOS app target.
```

Original comment by `mtu...@gmail.com` on 6 Oct 2015 at 10:56

**Comment by GoogleCodeExporter** [comment]
```
Out of all the companies' analytics I could have chosen to integrate into my 
game, I would have guessed that Google would have been the least likely to not 
keep theirs functional. Is it abandoned for iOS? If it is, or if there is no 
plan to update it, just let us know so that we can rid ourselves of it. Please 
give us some idea what is up...
```

Original comment by `wjur...@gigabitgames.com` on 7 Oct 2015 at 2:46

**Comment by GoogleCodeExporter** [comment]
```
Google team, 
we all have application running their routines. We have responsibilities to all 
our customers, like you have with your (we Developer). 

Put themselves in our place and respond. 

What would you say to a customer standing with his application stop the day?

What would you say? 

Please we need a forecast to move on to our customers.
```

Original comment by `mrom...@gmail.com` on 7 Oct 2015 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
For those people that were experiencing this issue in conjunction with Xcode 
7's code signing bug (caused by bitcode disabled in the parent, enabled in the 
child), I'm happy to report that after working through the Xcode issue via a 
radar, last night Apple instructed me to test against Xcode 7.1 beta 3, and 
that issue is resolved for me. I'd still like Google analytics to support bit 
code, but I am finally able to distribute via enterprise distribution profiles 
again.
```

Original comment by `bran...@acorns.com` on 7 Oct 2015 at 5:06

**Comment by GoogleCodeExporter** [comment]
```
I test with Xocde 7.1 beta 3, but not work with me.
Waiting Google Team.
```

Original comment by `mrom...@gmail.com` on 9 Oct 2015 at 2:36

**Comment by GoogleCodeExporter** [comment]
```
We thank everyone for their patience as we are actively working to get the new 
SDK released as soon as possible. As work around it is possible to build and 
deploy projects that have the current Google Analytics SDK with bitcode 
disabled for iOS targets and with watchOS targets with bitcode enabled.

There is an example on stackoverflow on how to disable bitcode.
http://stackoverflow.com/questions/30848208/new-warnings-in-ios9

There there are also reports that the related xcode distribution profile 
signing issue for applications with this configuration is resolved with Xcode 
7.1 beta 3:
https://forums.developer.apple.com/thread/9734


```

Original comment by `mcoh...@google.com` on 10 Oct 2015 at 2:20

**Comment by GoogleCodeExporter** [comment]
```
@mcoh

Thank you for the heads up. :)
```

Original comment by `scarn...@gmail.com` on 10 Oct 2015 at 5:35

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the heads up.
```

Original comment by `da...@imaggle.com` on 11 Oct 2015 at 5:25

**Comment by GoogleCodeExporter** [comment]
```
!As work around it is possible to build and deploy projects that have the 
current Google Analytics SDK with bitcode disabled for iOS targets and with 
watchOS targets with bitcode enabled."

You can't, if you have another framework with bitcode enabled, for example 
google maps sdk ;-)
How long does it take to set a flag, rebuild and publish?
```

Original comment by `jwolff....@googlemail.com` on 13 Oct 2015 at 8:35

**Comment by GoogleCodeExporter** [comment]
```
Hello jwolf,
Thank you for writing in.

Integrating with one library that has bitcode doesn't make it a requirement for 
your entire application. As far as I am aware the Google Maps SDK has not yet 
enabled bitcode: 
https://code.google.com/p/gmaps-api-issues/issues/detail?id=8219

Also, I will update this thread with any news about upcoming release.
```

Original comment by `mcoh...@google.com` on 13 Oct 2015 at 11:03

**Comment by GoogleCodeExporter** [comment]
```
@mcoh: The problem is, that bitcode enabled frameworks are not delivered as a 
fat library, including all the architectures you need. And you are right, 
Google Maps is not bitcode enabled yet. I looked at the demo project, which was 
bitcode enabled because this is the XCode 7 dafault setting.

Thank you for updating :-)
```

Original comment by `jwolff....@googlemail.com` on 14 Oct 2015 at 1:51

**Comment by GoogleCodeExporter** [comment]
```
@mcoh: Thanks for updating.
```

Original comment by `srf.tobi...@gmail.com` on 14 Oct 2015 at 11:19

**Comment by GoogleCodeExporter** [comment]
```
Is there any known workaround to release iOS app with bit code enabled that is 
using Google Analytics?

With the latest cocoa pod for Google/Analytics (1.1.1) we are getting Xcode 
linker error when building for iOS Device:

...GoogleUtilities/Libraries/libGTM_NSData+zlib.a(GTMNSData+zlib.o)' does not 
contain bitcode.
You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE)...
```

Original comment by `pzabe...@gmail.com` on 18 Oct 2015 at 8:42

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Is there any particular reason why it takes so log to fix these issue. It 
should be:
 just open it and rebuild with new Xcode. Why does it took 4 month
"issue openend - Jun 18, 2015"
```

Original comment by `kostian...@rocketfarm.no` on 21 Oct 2015 at 11:27

**Comment by GoogleCodeExporter** [comment]
```
@mcoh Can you give us a little more insight into this delay? It doesn't make 
much sense to most developers. My guess is that you guys are following an 
internal process that makes it difficult to release patches? Do you commit 
directly into master, which might explain this delay?  Did a major point 
release get caught in a merge process that is now making a simple rebuild take 
months? 

I understand why arm64 may have been a lot more involved because of very real 
migration issues, but the bitcode delay is still very alarming. Are you facing 
issues with Apple's toolchain? Is there some edge case, specific to how this 
library was implemented, that is making you jump through hoops? 

Any information would be great. 
```

Original comment by `RRomanc...@gmail.com` on 21 Oct 2015 at 8:00

**Comment by GoogleCodeExporter** [comment]
```
Hi Roman,
There are a number reasons for the delay, but largely it is because the 
underlying build infrastructure needed to be updated to support Xcode 7 while 
still maintaining support for legacy versions. And that change required a whole 
chain of dependencies to be fixed. There are also stringent quality assurance 
and testing procedures that ensure all the libraries will work once released. 
The team responsible for cutting the final release has assured me that a 
release is coming soon.
Matt
```

Original comment by `mcoh...@google.com` on 21 Oct 2015 at 11:54

**Comment by GoogleCodeExporter** [comment]
```
Thank you!

Will Larche
Chief Product Officer, Lead iOS Engineer
Miner Labs, Inc. 

Download Miner on the App Store <http://apple.co/1J3wy2O>
```

Original comment by `w...@minerapp.com` on 22 Oct 2015 at 12:25

**Comment by GoogleCodeExporter** [comment]
```
Got it, 
Thanks Matt!  God speed QA team :P 
```

Original comment by `RRomanc...@gmail.com` on 22 Oct 2015 at 12:34

**Comment by GoogleCodeExporter** [comment]
```
How can I prevent from someone tracking my accounts or moving ads that I never 
got into? I don't know much about computers but I do know someone is using my 
email, facebook,twitter &etc.
```

Original comment by `rgakittie@gmail.com` on 22 Oct 2015 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
Good news everyone, we have just released v3.14 with bitcode support:


https://developers.google.com/analytics/devguides/collection/ios/changelog

https://developers.google.com/analytics/devguides/collection/ios/v3/sdk-download
```

Original comment by `mcoh...@google.com` on 23 Oct 2015 at 6:47

**Comment by GoogleCodeExporter** [comment]
Original comment by `mcoh...@google.com` on 23 Oct 2015 at 6:49
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Yaaay! Hit that close button, and head to the bar! 
```

Original comment by `RRomanc...@gmail.com` on 23 Oct 2015 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
Hi Matt!

Now that this has been solved for 4 whole minutes, can you please also update 
the Cocoapods trunk repo for this as well? :)

It looks like (as of ~noon Pacific time on Friday, October 23rd) the Google 
Analytics repo is still using version `3.1.13`.

https://cocoapods.org/pods/Google
https://cocoapods.org/pods/GoogleAnalytics

THANKS!

-sean
```

Original comment by `tobler...@gmail.com` on 23 Oct 2015 at 6:55

**Comment by GoogleCodeExporter** [comment]
```
A note about the cocoapods: "GoogleAnalytics" will be updated soon with this 
latest release, and if you are using that directly you should be able to enable 
bitcode. But if you are using the "Google" pod, the top level library does not 
yet have bitcode enabled. But it is being worked on now as well.
```

Original comment by `mcoh...@google.com` on 23 Oct 2015 at 7:07

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
How soon is "soon"?
```

Original comment by `james.bi...@cuvva.co` on 26 Oct 2015 at 8:55

**Comment by GoogleCodeExporter** [comment]
```
Yeah, we need cocoapods!
```

Original comment by `rusya....@gmail.com` on 27 Oct 2015 at 3:06

**Comment by GoogleCodeExporter** [comment]
```
Still waiting :(
```

Original comment by `palka.pa...@gmail.com` on 28 Oct 2015 at 7:48

**Comment by GoogleCodeExporter** [comment]
```
yeah still waiting...
```

Original comment by `felipe.m...@gmail.com` on 28 Oct 2015 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
Still waiting...
```

Original comment by `Artur.Ba...@sflpro.com` on 29 Oct 2015 at 7:28

**Comment by GoogleCodeExporter** [comment]
```
We are about to release a new version of our app an it would be great if we 
could support Bitcode via Cocoapods. This is the only library left now so can 
we get an update on a new Cocoapods release? 
```

Original comment by `gwynantj...@gmail.com` on 29 Oct 2015 at 12:31

**Comment by GoogleCodeExporter** [comment]
```
I Tried to update without any luck
https://github.com/CocoaPods/Specs/blob/master/Specs/GoogleAnalytics/3.13.0/Goog
leAnalytics.podspec.json

Whats weird :
- Dont' know where that archive referenced in podspec is coming from : (3.13)
  https://www.gstatic.com/cpdc/9b1bb5e186325dc2-GoogleAnalytics-3.13.0.tar.gz (814k)
- Archive : 3.13 SDK is 6.9 MB and the new 3.14 is 19.5 MB (what happened here 
?)
- libGoogleAnalyticsServices.a : 3.13 is 12.4 MB - 3.14 is 34.4 MB ! 
- Cocoapods 3.13 is using libGoogleAnalytics.a which doesn't exists in their 
SDK's probably because of "This is a combined SDK that includes both the Google 
Analytics SDK
and the Google Tag Manager SDK."

```

Original comment by `arso...@gmail.com` on 29 Oct 2015 at 2:10

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Just uploaded a new version of my app to the App Store... still no Bitcode for 
CocoaPods... #waiting
```

Original comment by `rgome...@gmail.com` on 29 Oct 2015 at 4:39

**Comment by GoogleCodeExporter** [comment]
```
google, Like my grandma used to tell me "If your software is not available for 
cocoapods or carthage, then your software has not been released."   
```

Original comment by `RRomanc...@gmail.com` on 31 Oct 2015 at 5:33

**Comment by GoogleCodeExporter** [comment]
```
Please add this version to cocoa pods repo - thnx
```

Original comment by `wie...@gmail.com` on 2 Nov 2015 at 3:00

**Comment by GoogleCodeExporter** [comment]
```
+1 Please add the new version to cocoapods repo.
```

Original comment by `neil.a.d...@gmail.com` on 3 Nov 2015 at 12:50

**Comment by GoogleCodeExporter** [comment]
```
Issue 774 has been merged into this issue.
```

Original comment by `mcoh...@google.com` on 3 Nov 2015 at 5:32

**Comment by GoogleCodeExporter** [comment]
```
I will re-open this issue until the Google Analytics Cocoapod with 3.14 is 
released. We anticipate this to be made available in the coming weeks. I cannot 
give any concrete timeline but this is something we are actively working on.

Keep in mind that for iOS application bitcode is still optional. And if you are 
using the top level Google Cocoapod with any libraries that have yet to support 
bitcode you will still need to disable bitcode for you iOS application.

I will update once this thread once the Cocoapod  with the 3.14 library is 
available. We appreciate your patience. 
```

Original comment by `mcoh...@google.com` on 3 Nov 2015 at 5:33
- Changed state: **Prioritized**

**Comment by GoogleCodeExporter** [comment]
```
is not optional if you support watchOS... I can't release my app because of 
this library... :(
```

Original comment by `arielaaa...@gmail.com` on 3 Nov 2015 at 7:11

**Comment by GoogleCodeExporter** [comment]
```
As long as you don't include the google analytics library in your watchOS 
target, you can leave bitcode on for the watch. Then on the iOS target turn off 
bitcode.

This way you can submit your app which includes watch and analytics in the 
phone.
```

Original comment by `mcoh...@google.com` on 3 Nov 2015 at 7:43

**Comment by GoogleCodeExporter** [comment]
```
ohh!! thanks!
```

Original comment by `arielaaa...@gmail.com` on 3 Nov 2015 at 8:24

**Comment by GoogleCodeExporter** [comment]
```
The changlog only mentions "Added support for bitcode." but the download size 
increased from 6.9MB to 19.5MB?  The library goes from 12.4MB to 34.4MB?  That 
feels like a large jump.  Why is it so large?
```

Original comment by `wredskin...@gmail.com` on 3 Nov 2015 at 10:37

**Comment by GoogleCodeExporter** [comment]
```
Hello W,
This is due to bitcode. The increase in size is due to the fact that bitcode is 
an intermediate (not-fully compiled) representation of the code. Developers 
will see this increase in their own app code as well with bitcode enabled, 
regardless of what libraries you include.

However, what the end user downloads from the app store is much smaller. Apple 
does not ship the bitcode to users.

For comparison, the Paypal SDK went from 12MB to 120MB  with bitcode enabled 
(https://github.com/paypal/PayPal-iOS-SDK/releases).

```

Original comment by `mcoh...@google.com` on 4 Nov 2015 at 10:16

**Comment by GoogleCodeExporter** [comment]
```
Issue 781 has been merged into this issue.
```

Original comment by `mcoh...@google.com` on 5 Nov 2015 at 4:13

**Comment by GoogleCodeExporter** [comment]
```
Just want to make a quick comment here.  The delays for critical features are 
unacceptable.  As a developer and founder, I'm not leaving any chance when it 
comes to getting my features out the door.  I decided to abandon Google 
Analytics for Flurry, and it has been smooth sailing since.
```

Original comment by `mjgroves...@gmail.com` on 5 Nov 2015 at 11:01

**Comment by GoogleCodeExporter** [comment]
```
Please add bitcode support. This is not acceptable, since bitcode is now a 
requirements from Apple:

XCode 7.1 Release Notes - Bitcode:

Bitcode

Apps that include incorrectly-built bitcode in frameworks and libraries are 
rejected by the App Store.
If you are a provider of framework and library products for iOS, watchOS, and 
tvOS platforms, your products must include full bitcode content. The Xcode 7 
build system defaults include enabling the Enable Bitcode build setting, but 
standard Debug or Release builds do not include the full bitcode content in the 
built framework and library products.

To build framework and library products for distribution and ensure that the 
full bitcode content is included, framework and library providers must:

Ensure that the Enable Bitcode build setting is correct (ENABLE_BITCODE=YES).
Perform either an Archive build or an Install build to produce framework and 
library products for distribution.

https://developer.apple.com/library/ios/releasenotes/DeveloperTools/RN-Xcode/Cha
pters/xc7_release_notes.html
```

Original comment by `lor...@musixmatch.com` on 9 Nov 2015 at 11:16

**Comment by GoogleCodeExporter** [comment]
```
Good news everyone. The Cocoapods have now been updated to use The Google 
Analytics 3.14 SDK which include bitcode support. Thank you everyone for your 
patience.

podspec: 
https://github.com/CocoaPods/Specs/blob/master/Specs/GoogleAnalytics/3.14.0/Goog
leAnalytics.podspec.json


```

Original comment by `mcoh...@google.com` on 10 Nov 2015 at 10:20
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Newbie here, I'm still getting the Bitcode error after updating to 3.14.0 (via 
pod update):

ld: '/Pods/GoogleUtilities/Libraries/libGTM_NSData+zlib.a(GTMNSData+zlib.o)' 
does not contain bitcode. You must rebuild it with bitcode enabled (Xcode 
setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable 
bitcode for this target. for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

And my pod file:

pod "Google/Analytics"

And pod update:

Update all pods
Updating local specs repositories
Analyzing dependencies
Using Google (1.2.1)
Using GoogleAnalytics (3.14.0)
Using GoogleInterchangeUtilities (1.1.0)
Using GoogleNetworkingUtilities (1.0.0)
Using GoogleSymbolUtilities (1.0.3)
Using GoogleUtilities (1.1.0)

Did i do something wrong?
```

Original comment by `zenk...@gmail.com` on 10 Nov 2015 at 11:13

**Comment by GoogleCodeExporter** [comment]
```
That is because you are using top level pod "Google/Analytics".  Which is tied 
to the state of greater Google pod. See comment 96 above. If you are only using 
the Google Analytics  component of that pod you should be able to use rebuild 
with GoogleAnalytics https://cocoapods.org/pods/GoogleAnalytics.

```

Original comment by `mcoh...@google.com` on 10 Nov 2015 at 11:26

**Comment by GoogleCodeExporter** [comment]
```
Ah I see. I saw the comment 96 but wasn't aware the difference between 
"Google/Analytics" and "GoogleAnalytics". Silly me.

Thanks for the quick reply.
```

Original comment by `zenk...@gmail.com` on 10 Nov 2015 at 11:31

**Comment by GoogleCodeExporter** [comment]
```
You will also have to follow the initialization method listed here... 
https://developers.google.com/analytics/devguides/collection/ios/v2/     

The latest instructions are going to give you a headache because it references 
utilities not included in the pod.  Below will fail with the cocoapod

// Configure tracker from GoogleService-Info.plist.
NSError *configureError;
[[GGLContext sharedInstance] configureWithError:&configureError];
NSAssert(!configureError, @"Error configuring Google services: %@", 
configureError);
```

Original comment by `RRomanc...@gmail.com` on 10 Nov 2015 at 11:32

**Comment by GoogleCodeExporter** [comment]
```
That's super and confirmed:

macbookproloreto:GoogleAnalytics admin$ otool -l libGoogleAnalyticsServices.a | 
grep __LLVM

 segname __LLVM
   segname __LLVM
   segname __LLVM
   segname __LLVM
   segname __LLVM
   segname __LLVM


(Read here about Bitcode and Assembly: 
https://forums.developer.apple.com/thread/3971 )

Now, we need tvOS TARGET!

```

Original comment by `lor...@musixmatch.com` on 11 Nov 2015 at 8:53

**Comment by GoogleCodeExporter** [comment]
```
Mhm. I've imported Google/Analytics before. Now using pod 'GoogleAnlaytics' but 
get a lot of errors. Is there any description available on how to configure a 
Swift Project correctly if I just use the Analytics Package withou all the 
other stuff?
```

Original comment by `christia...@instana.com` on 13 Nov 2015 at 1:08

**Comment by GoogleCodeExporter** [comment]
```
E.g GAIDictionaryBuilder will not be found if I only use GoogleAnalytics 
Package via pods
```

Original comment by `christia...@instana.com` on 13 Nov 2015 at 1:12

**Comment by GoogleCodeExporter** [comment]
```
Will you be also updating the top level Google/analytics pod ? Because i am 
getting error on GGlContext 
```

Original comment by `basty191...@gmail.com` on 16 Nov 2015 at 1:32

**Comment by GoogleCodeExporter** [comment]
```
I also need the bitcode enabled in the Google/Analytics pod. When will it be 
updated?
```

Original comment by `llerena....@gmail.com` on 22 Nov 2015 at 5:35

**Comment by GoogleCodeExporter** [comment]
```
Having trouble implementing this in swift because of the lack of bitcode 
support. Having trouble using base analytics pod. 
```

Original comment by `template...@gmail.com` on 8 Dec 2015 at 8:38

**Comment by GoogleCodeExporter** [comment]
```
Google/Analytics needs bitcode support 
```

Original comment by `r...@brighten.in` on 20 Jan 2016 at 9:33

**Comment by GoogleCodeExporter** [comment]
```
Agrh google, not you again!
```

Original comment by `Dincho.T...@gmail.com` on 25 Jan 2016 at 2:10

**Comment by GoogleCodeExporter** [comment]
```
In local it works fine, but I have a problem in Jenkins. It takes hours to 
figured it out

Replace this in your pod file
pod 'GoogleAnalytics'

In your Bridging-Header.h (I use Swift)
#import "GAI.h"
#import "GAIDictionaryBuilder.h"
#import "GAIFields.h"

Add User headers search paths for both build and test target
$(SRCROOT)/Pods/Headers with recursive

Now I can sleep
```

Original comment by `nguyen.c...@gmail.com` on 27 Jan 2016 at 4:03

**Comment by GoogleCodeExporter** [comment]
```
Work around with GoogleAnalytics pod along with a Bridging Header to import the 
headers works but it's not perfect.  For instance now GA needs to be compiled 
along with any test targets too.

Can we please get an update on the actual Google/Analytics pod to use bitcode 
already?  It's been over 4 months since the official release of Xcode 7 and 
Bitcode was announced in beta about 7 months ago.  
```

Original comment by `chris.ch...@303software.com` on 28 Jan 2016 at 7:26

**Comment by GoogleCodeExporter** [comment]
```
Any update on bit code support in the library ? It's been a long time since 
Xcode 7 came out. 
```

Original comment by `i...@thelocalite.in` on 2 Feb 2016 at 8:33

**Comment by GoogleCodeExporter** [comment]
```
If you are using cocoapods, you can add this at the end of the Podfile:

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
    end
  end
end

This will recurse through all the pods and mark the ENABLE_BITCODE off. 
Naturally, you will not be able to compile for the watch / apple tv (as they 
require it on some versions).
```

Original comment by `luis.fer...@gmail.com` on 4 Feb 2016 at 5:18
